docs/docs/vulnerability/scanning/git-repository.md

```diff
@@ -188,6 +190,11 @@ $ export GITLAB_TOKEN="your_private_gitlab_token"
 $ trivy repo <your private GitLab repo URL>
 ```
 
+Note that if the Trivy scan could not identify any vulnerabilities, you can try Trivy misconfiguration scanning instead with the following flag:
```

Comment:
I don't get the point. Why do we recommend detecting misconfigurations when no vulnerabilities are found? I think the motivation of detecting misconfiguration is irrelevant to vulnerabilities, no?

Reply:
hmm good point, not sure what I was thinking lol